css/style.css

```diff
@@ -20,7 +20,19 @@
     min-height: 632px;
     max-height: 864px;
     position: relative;
-    background-color: #77c159;
+    background-image: url("../img/madrid.jpg");
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: cover;   
+}
+
+.overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(119,193,89,0.5);  
```

Comment:
It's the same that @SADGevents/valencia, but if you want, when you check the alpha, change it. 

alpha 0.5: 
![image](https://cloud.githubusercontent.com/assets/5845622/18069841/fc752b68-6e49-11e6-9881-5e5f3a8fbf66.png)

alpha 0.8:

![image](https://cloud.githubusercontent.com/assets/5845622/18069863/1e4a9c28-6e4a-11e6-9338-2f873ee28e89.png)